developer.md

```diff
@@ -313,6 +344,25 @@ Aspects of Perma's paid subscription service are handled by the companion applic
 
 By default, Perma's `docker-compose` file will spin up a local Perma Payments for you to experiment with. For more fruitful experimentation, configure this Perma Payments to interact with Cybersource's test tier, by running Payments with a custom settings.py that contains our credentials. See `docker-compose.yml` and `/services/docker/perma-payments/settings.py.example` for more information. CyberSource will not be able to communicate its responses back to your local instance, of course, but you can simulate active subscriptions using the Django admin.
 
+### Test Perma Interaction with Perma Payments
+
+You may also decide to run the two services locally to experiment by running both on the same network.
+
+The Docker containers for `perma-payments` have to be built before you can successfully run  `perma` on the same network. Visit the [`perma-payments` repo](https://github.com/harvard-lil/perma-payments/blob/develop/README.md#running-locally) for those instructions. Once you've done that, build the Docker containers for perma: 
+
+- `docker compose -f docker-compose.yml up -d --build`
+
+Then, to run perma, run:
+
+`d fab run`
+
+When you're finished, take down the containers by running:
+
+`docker compose -f docker-compose.yml down`
+
+Don't worry if you get the following error when shutting down these containers while still running the `perma-payments` containers. That's because the network is maintained until both containers are down:
```

Comment:
```suggestion
Don't worry if you get the following error when shutting down these containers while still running the `perma-payments` containers. That's because the network is maintained until both Docker Compose projects are down:
```